cucumber-jvm/src/org/jetbrains/plugins/cucumber/java/CucumberJavaExtension.java

```diff
@@ -31,6 +31,8 @@
     static final String ZUCHINI_RUNTIME_JAVA_STEP_DEF_ANNOTATION = "org.zuchini.annotations.StepAnnotation";
     static final String CLASS_NAME_OPEN_PAGE = "openPage";
     static final String CLASS_NAME_CORE_STEP_DEFS = "CoreStepDefs";
+    static final String CLASS_NAME_SEND_REQUEST = "send";
+    static final String CLASS_NAME_API_STEPS_IMPL = "ApiStepDefs";
```

Comment:
maybe it's better to deal with fqdn classnames?

Reply:
Guess no, because fqdn contains language part, like '.ru.CoreStepDefs', and this class isn't an extension of ru.sbtqa.tag.pagefactory.junit.CoreSteps